### ISSUE TITLE
feature request: info about currently targeted OF install

### ISSUE THREAD

**Issue by artificiel** [openframeworks/projectGenerator#435]
when using PG while juggling different OF trees it sometimes gets confusing to know which OF is actually being target. 

proposal to add a "status line" indicating the current OF selection on the main window:

![image](https://github.com/openframeworks/projectGenerator/assets/2172370/45f7367b-2b5b-43ca-94bc-390d84991526)

**Comment by danoli3** [comment]
Agreed its easy to mess this up as even if you run projectGenerator, a different one from a different checkout it may go to the last used path.

Also this maybe a bug maybe not taking into account in the config/save file where the context for this current projectGenerator is being used from.

C:/openFrameworksOriginal/projectGenerator.app - > used first
C:/openFrameworksSideQuestCheckout/projectGenerator.app -> used second will automatically be set to "C:/openFrameworksOriginal/" as the known checkout. 

Transparency of this path thus, a good idea first and second lets fix that config save bug

**Comment by danoli3** [comment]
v 0.96.0 has this at the top
